definitions/test/fixtures/steps/upload.response.ts

```diff
@@ -86,8 +87,9 @@ export class UploadResponse extends BaseStep {
 
         await this.homePage.navigateToTab("Summary");
         await this.summaryTab.verifyPresenceOfText("Ready to list");
-        
-        await this.page.locator('button.mat-tab-header-pagination-after').click();
+        if (environment.name == 'aat') {
```

Comment:
Errr, what's this?

Reply:
This is because the Panels and Hearings tab is Showing and extra Panel and Hearings tab and the Preview is not.
This should be investigated as to why there is a difference in the relevant environments but for now as that issue is not related to this card.

A tactical fix has been put in for the tests to pass.Discussed this issue with Makeda and she wanted to raise it with you.I will raise the defect after know the cause and reason.